docs/usage.md

```diff
@@ -53,12 +54,22 @@ sample1,0,data/sample1.fastq.gz,,
 sample2,0,data/sample2.fastq.gz,,
 ```
 
+or to additionally to perform run merging of two runs of sample1:
+
+```bash
```

Comment:
The documentation above mentions using a `run` column, but the example here excludes it - is it accidentally misnamed `group`? (it varies within the sample). Does group need to be additionally specified as well?

Reply:
Oops, good call. Forgot to include the _additional_ run column